src/components/about-us/opening/background.js

```diff
@@ -0,0 +1,201 @@
+import { intro } from '../constants/data/introduction'
+import { screen } from '../utils/screen'
+import billboard from '../../../../static/asset/about-us/billboard.png'
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+
+const triangleSizes = {
+  hd: {
+    width: '653px',
+    height: '112px'
+  },
+  desktop: {
+    width: '464px',
+    height: '80px'
+  },
+  mobile: {
+    width: '43px',
+    height: '276px'
+  },
+  tablet: {
+    width: '102px',
+    height: '661px'
+  }
+}
+
+const DesktopTriangles = styled.div`
+  position: absolute;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  ${screen.tabletBelow`
+    display: none;
+  `}  
+`
+
+const MobileTriangles = styled.div`
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  right: 0;
+  ${screen.desktopAbove`
+    display: none;
+  `}
+`
+
+const Triangle = styled.div`
+  ${screen.desktop`
+    width: ${triangleSizes.desktop.width};
+    height: ${triangleSizes.desktop.height};
+    border-top: solid ${triangleSizes.desktop.height} #c7000a;
+    border-left: solid calc(${triangleSizes.desktop.width} * (3 / 8)) transparent;
+    border-right: solid calc(${triangleSizes.desktop.width} * (5 / 8)) transparent;  
+  `}
+  ${screen.overDesktop`
+    width: ${triangleSizes.hd.width};
+    height: ${triangleSizes.hd.height};
+    border-top: solid ${triangleSizes.hd.height} #c7000a;
+    border-left: solid calc(${triangleSizes.hd.width} * (3 / 8)) transparent;
+    border-right: solid calc(${triangleSizes.hd.width} * (5 / 8)) transparent;
+  `}  
+`
+
+const TriangleRightTop = Triangle.extend`
+  position: absolute;
+  right: 0;
+  top: 0;
+`
+
+const TriangleRightBottom = Triangle.extend`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  transform: rotateX(180deg);  
+`
+
+const MobileTriangle = styled.div`
+  width: ${triangleSizes.mobile.width};
+  height: ${triangleSizes.mobile.height};
+  border-left: solid ${triangleSizes.mobile.width} #c7000a;
+  border-top: solid calc(${triangleSizes.mobile.height} * (3 / 8)) transparent;
+  border-bottom: solid calc(${triangleSizes.mobile.height} * (5 / 8)) transparent;
+  ${screen.tablet`
+    width: ${triangleSizes.tablet.width};
+    height: ${triangleSizes.tablet.height};
+    border-top: solid calc(${triangleSizes.tablet.height} * (3 / 8)) transparent;
+    border-bottom: solid calc(${triangleSizes.tablet.height} * (5 / 8)) transparent;
+  `}      
+`
```

Comment:
`Triangle` and `MobileTriangle` seems to be duplicate.
Could you refactor it to a function with different arguments?
Like 
```
function GetTriangleStyles(device='mobile') {
  return `
    border-left: solid ${triangleSizes[device].width} ${colors};
    ...
  `
}

const MobileTriagnle = styled.div`
${GetTriangleStyles(mobile)}

${screen.tablet`
  ${GetTriangleStyles(tablet)}
`}
`
```